src/SFA.DAS.RoatpOversight.Web/Services/ApplicationOutcomeOrchestrator.cs

```diff
@@ -48,8 +51,11 @@ public async Task<bool> RecordOutcome(Guid applicationId, bool? approveGateway,
             if (outcome == OversightReviewStatus.Successful)
             {
                 var request = BuildCreateOrganisationRequest(userName, registrationDetails);
+             
 
-                await _registerApiClient.CreateOrganisation(request);
+                var createOrganisationResponse= await _registerApiClient.CreateOrganisation(request);
+                if (createOrganisationResponse)
+                     await AddProviderToRoatpCourseManagement(registrationDetails,userId, userName, request.ProviderTypeId);
```

Comment:
Prefer to have curly braces here unless you want to change it to be on the same line